stories/imdb/stories/searchButton.js

```diff
@@ -11,14 +11,15 @@ import {
   componentForType,
   FilterAdder,
 } from '../../../src'
-import { Button, Pager, ExampleTypes } from '../../DemoControls'
+import theme, { Button } from '../../DemoControls'
+import ExampleTypes, { TypeMap } from '../../../src/exampleTypes'
```

Comment:
imports

Reply:
also this whole thing could be a trivial alternate theme (especially if we do an IMDB kitchen sink with a theme picker)